pkg/devfile/header.go

```diff
@@ -22,4 +26,6 @@ type DevfileMetadata struct {
 	// +optional
 	// +kubebuilder:validation:Pattern=^([0-9]+)\.([0-9]+)\.([0-9]+)(\-[0-9a-z-]+(\.[0-9a-z-]+)*)?(\+[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$
 	Version string `json:"version,omitempty"`
+
+	attributes.Attributes `json:",inline"`
```

Comment:
Should it be `json:"attributes,omitempty"`

Reply:
I was trying to have the free-form attributes at the same level as the main metadata, but this is obviously not really possible.
I finally changed it according to your suggestion in my last commit:  https://github.com/devfile/api/pull/214/commits/1a8f497e4106b5c5b23b61d020b0d9df4799e455